src/main/resources/scripts/simple motd.sk

```diff
@@ -1,23 +0,0 @@
-#
-# A simple message of the day script.
-# The MOTD can be set with the /setmotd command,
-# and it will be sent to each player when they log in
-# and when they use the /motd command.
-#
-
-command /setmotd <text>:
-	permission: skript.setmotd
-	description: Sets message of the day
-	trigger:
-		message "Set the MOTD to '%argument%<reset>'"
-		set {motd} to argument
-
-command /showmotd:
-	description: Displays the message of the day
-	trigger:
-		message {motd}
-
-on join:
-	# uncomment the following line to make the MOTD appear after other messages
-	# wait a tick
-	message {motd}

```

Comment:
Why is this removed?

Reply:
It is not removed, just remade here: https://github.com/SkriptLang/Skript/pull/2559/files#diff-18e684da942462e3df6b8fc506c3aa90

Reply:
Join messages/Quit messages are different from MOTD's

MOTD's are sent to a player only when they join, join/quit messages are sent to everyone when a player joins/leaves.

Reply:
Is it not a useless example? I mean, in 2 lines you can do it:
```vb
on join:
    send "Welcome back %player%!"
```
Seems useless to add an example for that.

Reply:
Those are examples to teach something to the users, not snippets. Anyways it is still an useful system, and the code teaches the Skript user useful things.